yt/yt/server/master/node_tracker_server/node_tracker.cpp

```diff
@@ -1209,6 +1296,8 @@ class TNodeTracker
                 ? std::make_optional(TYsonString(request->cypress_annotations(), EYsonType::Node))
                 : std::nullopt,
             .BuildVersion = request->has_build_version() ? std::make_optional(request->build_version()) : std::nullopt,
+            .Rack = request->has_rack() ? std::make_optional(request->rack()) : std::nullopt,
+            .DataCenter = request->has_data_center() ? std::make_optional(request->data_center()) : std::nullopt,
```

Comment:
Ever since type deduction guides have appeared in the language, there's rarely a reason to use `make_optional` these day. `std::optional(request->rack())` will suffice.  

___

Also, there's a `YT_PROTO_OPTIONAL` macro that has been introduced comparatively recently. Seems convenient:

```cpp
.Rack = YT_PROTO_OPTIONAL(request, rack),
```